### ISSUE TITLE
FileNotFoundError 'logging.json'

### ISSUE THREAD

**Issue by barrystaes** [d0u9/youtube-dl-webui#16]
Looks great!

I tried starting the docker,
```
$docker run -d 
  --name="youtube-dl-webui" 
  --net="bridge" 
  -e TZ="Europe/Berlin" 
  -e HOST_OS="unRAID" 
  -e "PORT"="8680" 
  -e "PUID"="99" 
  -e "PGID"="100" 
  -e "CONF_FILE"="/config/config.json" 
  -p 8680:8680/tcp 
  -v "/mnt/user/appdata/youtube-dl-webui/":"/config/":rw 
  -v "/mnt/user/Downloads/youtube-dl-webui/":"/downloads/":rw 
  d0u9/youtube-dl-webui
```
but i see this error in the docker log output:
```
Traceback (most recent call last):
File "/usr/local/lib/python3.6/runpy.py", line 193, in _run_module_as_main
"__main__", mod_spec)
File "/usr/local/lib/python3.6/runpy.py", line 85, in _run_code
exec(code, run_globals)
File "/usr/src/youtube_dl_webui/youtube_dl_webui/__main__.py", line 20, in <module>
with open('logging.json') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'logging.json'
```

**Comment by d0u9** [comment]
I am trying to fix. Thanks for your feedback.

**Comment by d0u9** [comment]
Fixed in commit 13454cd.
